### DISCUSSION TITLE
climate fan only icon coloring enhancement

### DISCUSSION THREAD

**Discussion by astenger** [home-assistant/frontend#13924] [Theming]
The distinction of the fan symbol in the climate control for active vs inactive is very hard to see. For cooling mode the cooling icon is colored in blue, but for fan mode it is only a very small difference in shades of grey:

![image](https://user-images.githubusercontent.com/3413174/193186276-2db88fd8-843a-4f3b-8f85-60ffb3d3ce43.png)

It would be helpful if the active fan mode icon could be colored in something that jumps out a little more, e.g. green.

**Comment by technicalyaar** [comment]
it should be white

**Comment by ildar170975** [comment]
How does it look in DEFAULT light & dark themes?
Please post pics.

**Reply by astenger** [reply]
Thanks for pointing this out.

Default light theme:
![image](https://user-images.githubusercontent.com/3413174/193211407-ada71cb3-e254-4713-b69b-513323336823.png)

Looks like this:
![image](https://user-images.githubusercontent.com/3413174/193211433-2ff2927f-ae79-4749-bcc0-3b6e24a5a6c7.png)
The difference is slightly more visible, but the color would still help.

Default dark theme:
![image](https://user-images.githubusercontent.com/3413174/193211467-4bc24825-2b50-431a-9fe7-87623a13715c.png)

Looks like this:
![image](https://user-images.githubusercontent.com/3413174/193211504-6e5792b4-85ea-4c55-8245-b30f6dfaa206.png)
Pretty much as in the initial screen shot further above.

Hope this helps!

**Reply by astenger** [reply]
And just for reference, here are the default dark theme colors for "Heat":
![image](https://user-images.githubusercontent.com/3413174/193221514-5493cee1-501a-4273-a310-cb1dc30b0ed3.png)

and "Dry":
![image](https://user-images.githubusercontent.com/3413174/193221557-992c5f08-44a6-4e62-8928-a4e92b7287ad.png)

and light theme colors for "Heat":
![image](https://user-images.githubusercontent.com/3413174/193221945-fe9e4976-5a88-4aaa-8be0-5bc1ab68feee.png)

and "Dry":
![image](https://user-images.githubusercontent.com/3413174/193222003-6c555f5f-5c9d-4705-9c13-32a6d1cf1f35.png)
